server/views/components/assessment/_assessment-area.njk

```diff
@@ -82,6 +80,25 @@
                                 <p>{{ locale.needScore.needScoreNotAvailable }}</p>
                             {% endif %}
                         {% endif %}
+
+                        {% if assessment.subData %}
+                            <br>
+                            <div class="assessment-score">
+                                <h3 class="govuk-heading-s">{{ locale.detail.lifestyleAndAssociatedNeedScore }}</h3>
+                                <p class="govuk-body">{{ assessment.subData.criminogenicNeedsScore }} out of {{ assessment.subData.upperBound }}. (Scores above
```

Comment:
Hate to be that guy buuuut, it looks like you've gone and switched to using the locale file - which is really good, but accidentally missed the `out of` string that is used in a few places 😅

Reply:
I kinda left that one cos the other PR is doing scores.

Reply:
I will accept this answer and put it on the other PR instead :)